presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergModule.java

```diff
@@ -167,6 +168,7 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(OrcFileWriterConfig.class);
 
         binder.bind(IcebergPlanOptimizer.class).in(Scopes.SINGLETON);
+        binder.bind(IcebergParquetDereferencePushDown.class).in(Scopes.SINGLETON);
```

Comment:
We probably don't need to bind the `IcebergParquetDereferencePushDown` here. Binding the plan optimizer looks sufficient.
Do you happen to have any specific reasons to bind it here?

Reply:
Both `IcebergParquetDereferencePushDown` and `IcebergPlanOptimizer` are implementation classes of `ConnectorPlanOptimizer`, so we should also bound `IcebergParquetDereferencePushDown` here.